packages/vulcan-forms/lib/components/Form.jsx

```diff
@@ -74,7 +75,7 @@ const getDefaultValues = convertedSchema => {
 
 const getInitialStateFromProps = (nextProps) => {
   const collection = nextProps.collection || getCollection(nextProps.collectionName);
-  const schema = collection.simpleSchema();
+  const schema = nextProps.schema ? new SimpleSchema(nextProps.schema) : collection.simpleSchema();
```

Comment:
Maybe the schema you pass should already be a SimpleSchema schema? It seems like it would simplify the component's logic.

Reply:
It would simplify this code, indeed, but it would be inconsistent with the way schemas are attached to the collections in Vulcan.
Unlike Meteor collections, where they are attached as `SimpleSchema` objects, in Vulcan they're passed as plain objects. 
So I think the API is more consistent if they stay plain objects everywhere

Reply:
Makes sense. Let's keep it like this then :)